tensorflow/tools/docker/Dockerfile.devel-cpu-mkl

```diff
@@ -0,0 +1,127 @@
+FROM ubuntu
+
+LABEL maintainer="Clayne Robison<clayne.b.robison@intel.com>"
+
+# It is possible to override these for releases.
+ARG TF_BRANCH=r1.4
+ARG BAZEL_VERSION=0.7.0
+ARG WHL_DIR=/whl
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+        build-essential \
+        curl \
+        git \
+        golang \
+        libcurl3-dev \
+        libfreetype6-dev \
+        libzmq3-dev \
+        pkg-config \
+        python-dev \
+        python-pip \
+        rsync \
+        software-properties-common \
+        unzip \
+        zip \
+        zlib1g-dev \
+        openjdk-8-jdk \
+        openjdk-8-jre-headless \
+        wget \
+	locate \
+	vim \
+        && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
+
+RUN pip --no-cache-dir install --upgrade \
+        pip setuptools
+
+RUN pip --no-cache-dir install \
+        ipykernel \
+        matplotlib \
+        numpy \
+        scipy \
+        sklearn \
+        pandas \
+        wheel \
```

Comment:
regular docker image also have jupyter stack installed, I think it's good idea to do it, that'll also mean EXPOSE 8888

Reply:
See above. It's not the purpose of this container.